transmitter.js

```diff
@@ -1,31 +1,19 @@
 let transmissionFailureCount = 0;
-
-function networkTransmitStub(celcius) {
-    console.log(`Temperature to transmit: ${celcius} celcius`);
-
-    // The intention here is to transmit the temperature over a network.
-    // However, this is a stub, so we just print the message above
-    // and give back a return code.
-
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // This stub always succeeds and returns 200
-    return 200;
+function networkTransmitStub(celsius) {
+  console.log(`Temperature to transmit: ${celsius} Celsius`);
+  const randomNumber = Math.random();
+  return randomNumber < 0.6 ? 200 : 500;
 }
 
-function transmitInCelcius(farenheit) {
-    const celcius = (farenheit - 32) * 5 / 9;
-    const returnCode = networkTransmitStub(celcius);
-    if (returnCode != 200) {
-        // non-ok response indicates failure while transmitting over the network
-        // let us keep a count of failures to report
-        // However, this code doesn't increment failures - that's a bug!
-        // Add a test below to catch this bug. Alter the stub above, if needed.
-        transmissionFailureCount += 0;
-    }
+function transmitInCelsius(fahrenheit) {
+  const celsius = ((fahrenheit - 32) * 5) / 9;
+  const returnCode = networkTransmitStub(celsius);
+  if (returnCode != 200) {
+    transmissionFailureCount++;

```

Comment:
You have fixed the issue by incrementing `transmissionFailureCount`. This was part of the next assignment, not this one.
Still... though you have fixed it, the test is failing. Why?